src/components/Answers.js

```diff
@@ -1,8 +1,16 @@
 import React from 'react';
 
-class Answers extends Component {
+const Answer = (props) => {
+  const { id, text } = props;
 
-  render() {
-    
-  }
-}
+  return(
+    <div>
+      <label>
+        <input type="radio" name={id} />
```

Comment:
You're on the right track here using a radio element and specifying a name attribute. The `name` attribute should actually be unique per question, rather than unique per answer. So for example, all of the answer elements in question 1 might have a name attribute equal to `answer-group-1`.